pkg/pillar/scripts/device-steps.sh

```diff
@@ -262,25 +263,63 @@ fi
 mkdir -p $DPCDIR
 
 # Look for a USB stick with a usb.json file
-# If found it replaces any build override file in /config
-# XXX note that filesystem on USB stick needs to be labeled with DevicePortConfig
-SPECIAL=$(cgpt find -l DevicePortConfig)
-if [ -n "$SPECIAL" ] && [ -b "$SPECIAL" ]; then
-    echo "$(date -Ins -u) Found USB with DevicePortConfig: $SPECIAL"
-    if ! mount -t vfat "$SPECIAL" /mnt; then
-        echo "$(date -Ins -u) mount $SPECIAL failed: $?"
-    else
+# XXX note that gpt on the USB stick needs to be labeled with DevicePortConfig
+# If there is a dump directory on the stick we put log and debug info
+# in there.
+# If there is an identity directory on the stick we put identifying
+# information in a subdir there.
+access_usb() {
+    # echo "$(date -Ins -u) XXX Looking for USB stick with DevicePortConfig"
+    SPECIAL=$(cgpt find -l DevicePortConfig)
+    if [ -n "$SPECIAL" ] && [ -b "$SPECIAL" ]; then
+        echo "$(date -Ins -u) Found USB with DevicePortConfig: $SPECIAL"
+        if ! mount -t vfat "$SPECIAL" /mnt; then
+            echo "$(date -Ins -u) mount $SPECIAL failed: $?"
+            return
+        fi
         keyfile=/mnt/usb.json
         if [ -f $keyfile ]; then
             echo "$(date -Ins -u) Found $keyfile on $SPECIAL"
             echo "$(date -Ins -u) Copying from $keyfile to $CONFIGDIR/DevicePortConfig/"
```

Comment:
I guess this print should change. DPCDIR is /var/tmp/zededa/DevicePortConfig

Reply:
Good catch.